junebug/tests/helpers.py

```diff
@@ -154,7 +154,14 @@ class JunebugTestBase(TestCase):
 
     default_channel_config = {
         'ttl': 60,
-        'amqp': {},
+        'amqp': {
+            'hostname': 'rabbitmq',
+            'vhost': '/',
+            'port': '5672',
+            'db': '0',
+            'username': 'guest',
+            'password': 'guest'
+        },
```

Comment:
I don't think we need to specify all of these parameters, since they already have defaults in the configuration file

https://github.com/praekelt/junebug/blob/e8e3e51710a578009cbebdb05048af4380bb5919/junebug/config.py#L32-L41